### DISCUSSION TITLE
Loading speed is very slow in react

### DISCUSSION THREAD

**Discussion by OlympuJupiter** [OlympuJupiter/astronomer-providers#2] [Q&A] [answered]
I created the small site by using react framework.
but the site speed was very slowly. I think the reason is that I used many useEffect.
this is right ?

**Comment by dtoyoda-dev** [comment, marked as answer]
In my humble opinion, that is not so good for your project.
because the site is rendered, when the developer use react hook.
so you have to reduce to hook as possible.
